ourrevolution/settings.py

```diff
@@ -86,6 +86,10 @@
     'EVENTS_PROMOTE_RECENT_CUTOFF_DAYS',
     14
 ))
+EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID = str(os.environ.get(
```

Comment:
`str` is not needed

Reply:
And actually it might break None handling so please remove?

Reply:
Done.